content/docs/0.8.x/operate/multi_cluster/index.md

```diff
@@ -0,0 +1,220 @@
+
+The important once that are used in the above example are:
+
+| Parameter                | Description             | Default        |
+| ------------------------ | ----------------------- | -------------- |
+| `distributor.stageFilter` | Sets the stage this service belongs to | `""` |
+| `distributor.serviceFilter` | Sets the service this service belongs to | `""` |
+| `distributor.projectFilter` | Sets the project this service belongs to | `""` |
+| `remoteControlPlane.enabled` | Enables remote execution plane mode | `false` |
+| `remoteControlPlane.api.protocol` | Used protocol (http, https) | `"https"` |
+| `remoteControlPlane.api.hostname` | Hostname of the control plane cluster (and port) | `""` |
+| `remoteControlPlane.api.apiValidateTls` | Defines if the control plane certificate should be validated | `true` |
+| `remoteControlPlane.api.token` | Keptn API token | `""` |
+
+
+## Troubleshooting
+
+### Execution plane service does not start working
+
+If you see in the Keptn Bridge that an event was triggered but no service was reacting upon this trigger, test the connectivity from the execution plane service to the control plane. (as mentioned above) The Helm Charts for the `helm-service` and `jmeter-service` have a built in sanity check that validates whether the connection to the control plane can be established. 
+
+<details><summary>Expand instructions</summary>
+<p>
+
+**Test (sanity check):**
+
+*Prerequisites:*
+
+* [helm CLI](https://helm.sh/docs/intro/install/)
+
+* Connect you to the cluster where the execution plane is running
+
+* For example, you want to test `jmeter-service` that is running in `keptn-exec` namespace, execute:
+
+  ```console
+helm test jmeter-service -n keptn-exec
+  ```
+
+* The expected outcome should be: 
+
+  ```console
+Pod jmeter-service-test-api-connection pending
+Pod jmeter-service-test-api-connection succeeded
+NAME: jmeter-service
+LAST DEPLOYED: Thu Feb 25 15:55:24 2021
+NAMESPACE: keptn-exec
+STATUS: deployed
+REVISION: 1
+TEST SUITE:     jmeter-service-test-api-connection
+Last Started:   Thu Feb 25 15:55:40 2021
+Last Completed: Thu Feb 25 15:55:42 2021
+Phase:          Succeeded
+  ```
+
+**Help:** 
+
+1. Validate the properties set in the `values.yaml`: 
+
+  ```
+remoteControlPlane:
+  enabled: true                         # < (1) set to true
+  api:
+    protocol: "http"                    # < (2) set protocol: http or https
+    hostname: ""                        # < (3) set Keptn hostname (without /api)
+    apiValidateTls: true                # < (4 - optional) option to skip TLS verification
+    token: ""                           # < (5) set Keptn API token
+  ```
+
+  - Is `enabled` set to `true`?
+  - Is the Keptn API endpoint on `http` or `https`? 
+  - Is the hostname of the Keptn API endpoint correct, e.g. `my.keptn-dev.company.com` (without `/api`)
+  - Do you want to skip TLS verification?
+  - Is the Keptn API token correct. (You can find it in the Keptn Bridge)
```

Comment:
or by following the guide for authenticating `../install/#authenticate-keptn-cli`

Reply:
I added it.